### DISCUSSION TITLE
Support for --config <key>=<value> Git option

### DISCUSSION THREAD

**Discussion by dekhaus** [gitpython-developers/GitPython#1724] [Q&A]
Hi

Its my understanding that support for the --config option was adding to GitPython in this PR ( https://github.com/gitpython-developers/GitPython/pull/836 )

I'd like to use GitPython to perform a 'clone_from' command and have that command be passed a <key>=<value> config option.  Based on what I've read in the notes for the aforementioned PR (836) - this option is supported.   

Assuming it is supported - I have no idea what the syntax is.   Here's an example of the complete Git command I want to emulate ...

git clone -c http.extraHeader='Authorization: Bearer __replace_with_some_token__'  https://replace_with_git_url.git

How would the above git command be replicated using GitPython ...

git.Repo.clone_from( ? ) 

?

Thanks
Dave

**Comment by Byron** [comment]
I recommend to try it and see what happens - if the command fails it will be shown as part of an exception. If it doesn't, it can be made visible by [setting `GIT_PYTHON_TRACE=1`](https://gitpython.readthedocs.io/en/stable/tutorial.html#git-command-debugging-and-customization).

Maybe that leads to the desired results.
